index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+    <body>
+        <div class="all">
+            <div class="top-container">
+                <div class="header">
```

Comment:
- [ ] You did great work for make the changes but there is one issue and the last one. Consider using the `<header>` tag instead of `<div>` to improve SEO for your Portfolio.